src/components/Entry/HmmModels/index.js

```diff
@@ -77,6 +94,52 @@ const HmmModelSection = ({ logo } /*: {logo: {}} */) => {
             data={JSON.stringify(logo)}
             processData={schemaProcessData}
           />
+          {details && (
+            <>
+              <h4>Profile HMM Information</h4>
+              <table className={f('light', 'table-sum')}>
+                <tbody>
+                  <tr>
+                    <td>HMM build commands</td>
+                    <td>
+                      Build method: {details.hmm?.commands?.build || ''}
+                      <br />
+                      Search method: {details.hmm?.commands?.search || ''}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>Gathering threshold</td>
+                    <td>
+                      Sequence:{' '}
+                      {details.hmm?.cutoffs?.gathering?.sequence || ''}
+                      <br />
+                      Domain: {details.hmm?.cutoffs?.gathering?.domain || ''}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>Download</td>
+                    <td>
+                      <Link
+                        href={`${config.root.API.href}/entry/pfam/${payload?.metadata?.accession}?annotation=hmm`}
+                        download={`${
+                          payload?.metadata?.accession || 'download'
+                        }.hmm.gz`}
+                      >
+                        <span
+                          className={f('icon', 'icon-common', 'icon-download')}
+                          data-icon="&#xf019;"
+                        />{' '}
+                        Download
+                      </Link>{' '}
+                      the raw HMM for this family
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </>
```

Comment:
I only now realize that the HMM download link won't be displayed if the details are not available.

Would it be possible to always show the Download row (with the link, as the first row of the table, and to show the two other rows (HMM commands and gathering thresholds) if they are available?

Reply:
Sure. The download URL also needs to be changed. It seems that it's always pointing to pfam. Changing that to

```jsx
<Link href={`${config.root.API.href}/entry/${payload?.metadata?.source_database}/${payload?.metadata?.accession}?annotation=hmm`} download={`${payload?.metadata?.accession || 'download'}.hmm.gz`}>
```